### PR TITLE
Update configure-server-endpoints.md

### PR DESCRIPTION
**Pull request by msbemba**
The Licensing page has been removed in product terms for Microsoft defender for endpoint server. Removed the hyperlink accordingly.

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10047

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy revirew

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Data collected by Defender for Endpoint is stored in the geo-location of the ten
 - Follow the onboarding steps for the corresponding tool
 
 >[!IMPORTANT]
->In order to be eligible to purchase Microsoft Defender for Endpoint Server SKU, you must have already purchased a combined minimum of any of the following, Windows E5/A5, Microsoft 365 E5/A5 or Microsoft 365 E5 Security subscription licenses.  For more information on licensing, see the [Product Terms](https://www.microsoft.com/licensing/terms/productoffering/MicrosoftDefenderforEndpointServer/all).
+>In order to be eligible to purchase Microsoft Defender for Endpoint Server SKU, you must have already purchased a combined minimum of any of the following, Windows E5/A5, Microsoft 365 E5/A5 or Microsoft 365 E5 Security subscription licenses.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-endpoints.md]:
```suggestion
>In order to be eligible to purchase Microsoft Defender for Endpoint Server SKU, you must have already purchased a combined minimum of any of the following: Windows E5/A5, Microsoft 365 E5/A5, or Microsoft 365 E5 Security subscription licenses.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@mjcaparas Pleas merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10093

**Comment by siosulli** [comment]
@msbemba - Do you think it could be updated to this: https://www.microsoft.com/en-us/licensing/product-licensing/products or will I go ahead and merge it? 

Thanks, Sinead.

**Comment by msbemba** [comment]
@siosulli I thought so too. But the original link talked about the product terms (specific to MDE for Servers) So I thought it is better to remove the link to reduce ambiguity as the link no longer exists.

**Pull request merged**